src/main/java/com/cloudbees/jenkins/plugins/BitBucketTrigger.java

```diff
@@ -156,7 +163,8 @@ public void writeLogTo(XMLOutput out) throws IOException {
 
         @Override
         public boolean isApplicable(Item item) {
-            return item instanceof AbstractProject;
+            return item instanceof Job && SCMTriggerItem.SCMTriggerItems.asSCMTriggerItem(item) != null
+                    && item instanceof ParameterizedJobMixIn.ParameterizedJob;
```

Comment:
@oleg-nenashev It was suggested by @jglick in a similar PR some days ago https://github.com/jenkinsci/github-plugin/pull/59#discussion_r34195953